### DISCUSSION TITLE
Δημιουργία ομάδων

### DISCUSSION THREAD

**Discussion by diogenisAl** [courses-ionio/sw#1031] [Announcements]
### Για να φτιάξετε την ομάδα σας (Προθεσμία τέλος 2ης εβδομάδα, δλδ Κυριακή 27 Φεβρουαρίου)

1. Η [δημιουργία και ο συντονισμός ομάδας](https://courses-ionio.github.io/help/team/) είναι μια δύσκολη τέχνη, για αυτό, εκτός από τις οδηγίες, σας δίνουμε αρκετό χρόνο και σχετικό πλαίσιο (=παράλληλη αποστολή εισαγωγής και βιογραφικού στις συζητήσεις της βοήθειας τις πρώτες δύο εβδομάδες, ώστε να γίνει η γνωριμία).
2. Για να δημιουργήσετε μια νέα ομάδα, αφού πρώτα φτιάξετε έναν [νέο λογαριασμό οργανισμού](https://docs.github.com/en/organizations), κάντε ένα νέο σχόλιο αντιγράφοντας τον παρακάτω πίνακα και περιμένετε να απαντήσουν στο σχόλιο σας.
3. Μπορείτε να εντοπίσετε και να καλέσετε νέα μέλη από [την εισαγωγή της 1ης εβδομάδας](https://courses-ionio.github.io/help/intro/) και τα [βιογραφικά της 2η εβδομάδας](https://courses-ionio.github.io/help/cv/) [που θα βρείτε στις συζητήσεις στο αποθετήριο της βοήθειας](https://github.com/courses-ionio/help/discussions/categories/show-and-tell)
4. Αν σας αρέσει κάποια ομάδα που υπάρχει ήδη στη συζήτηση παρακάτω, ρωτήστε αν μπορείτε να μπείτε και αν καμία ομάδα δεν είναι διαθέσιμη, τότε φτιάξτε μια νέα ομάδα.
5. Μην ξεχνάτε να ενημερώνετε τα στοιχεία των ομάδων σε περίπτωση αλλαγών και την κατάσταση της ομάδας, έτσι ώστε να φαίνεται πάντα ποιες είναι ενεργές

Η [δημιουργία και ο συντονισμός της ομάδας](https://courses-ionio.github.io/help/team/) είναι δική σας ευθύνη και θα γίνει σε νέο προφίλ οργανισμού που θα δημιουργήσετε στο γκιτχαμπ και θα γράψετε παρακάτω:

Μετά την δημιουργία της ομάδας αντιγράψτε και συμπληρώστε τον παρακάτω πίνακα στην πρώτη ανάρτηση της ομάδας σας παρακάτω:

```
Όνομα ομάδας: delta
GitHub organisation url: [delta](url)
Νέα μέλη δεκτά: ΝΑΙ/ΟΧΙ

| # | ΑΜ | GitHub profile |
| -- | -- | -- |
| 1 | 2003001 | gh link |
| 2 | 2003002 | gh link |
| 3 | ... | ... |
```

Παρακαλούμε χρησιμοποιήστε την παρακάτω περιοχή μόνο για την οργάνωση ομάδων και όχι για απορίες. 

Όλες οι απορίες γίνονται στο κεντρικό αποθετήριο βοήθειας:
http://github.com/courses-ionio/help/discussions/

**Comment by voltmaister** [comment]
Όνομα ομάδας : First Time Is The Charm
GitHub organisation url : [First Time Is The Charm](https://github.com/First-Time-is-the-Charm/Workplace)
Νέα μέλη δεκτά: ΟΧΙ

|#|Members |Id |Account link| presence (week3) |
|---|-------|-----------|-------|---|
|1|Voltmaister| P2019153|[Know me!](https://github.com/voltmaister)| |
|2|P19mark|P2019170|[Know me!](https://github.com/p19mark)| |
|3|NickLitharis|P2019083|[Know me!](https://github.com/NickLitharis)| |
|4|KonstantinosTourtsakis|P2019140|[Know me!](https://github.com/KonstantinosTourtsakis)| |
|5|odysseasp2019060|P2019060|[Know me!](https://github.com/odysseasp2019060/)| |
|6|Angeloth1|P2019095|[Know me!](https://github.com/Angeloth1)| |
|7|P19tsao|P2019032|[Know me!](https://github.com/p19tsao)| NO |

**Reply by KonstantinosTourtsakis** [reply]
Αίτημα εισαγωγής στην ομάδα.

Κωνσταντίνος Τουρτσάκης
ΑΜ: Π2019140

**Reply by p19tsao** [reply]
Επιθυμώ την ένταξη στην ομάδα 

Ον/νυμο : Τσαούσης Γεώργιος 
ΑΜ : Π2019032

**Reply by voltmaister** [reply]
> Έχει σταλεί σχετικό invite για την συμμετοχή σας στην ομάδα. Ευχαριστούμε!

**Reply by Angeloth1** [reply]
Επιθυμώ την ένταξη στην ομάδα 

Θώμος Άγγελος ( Π2019095 )

**Reply by kaxiti** [reply]
Αιτημα εισαγωγής σε αυτή την ομάδα

Οδυσσέας Οικονόμου
ΑΜ: Π2019060

**Reply by epidrome** [reply]
@voltmaister μπράβο για την ταχύτητα, μην ξεχάσετε να ενημερώσετε το 1ο ποστ με την τρέχουσα σύνθεση της ομάδας στον πίνακα που σας δίνουμε στην αρχική ανακοίνωση

**Reply by voltmaister** [reply]
Η ομάδα έχει κλείσει τις αποδοχές νέων ατόμων για την καλύτερη διαχείρηση.

**Reply by epidrome** [reply]
@voltmaister μπράβο, αλλά δεν έχετε ενημερώσει σχετικά το 1ο ποστ σας το οποίο γράφει ότι δέχεστε μέλη, επίσης, ποιοι είστε, δες λίγο τα πρώτα ποστ παρακάτω.

> μην ξεχάσετε να ενημερώσετε το 1ο ποστ με την τρέχουσα σύνθεση της ομάδας στον πίνακα που σας δίνουμε στην αρχική ανακοίνωση

**Reply by voltmaister** [reply]
Έγινε η σχετική διόρθωση.

**Reply by P2019030** [reply]
Επιθυμώ, αν και είμαι εκτός προθεσμίας, να γίνει δεκτό το αίτημα μου για εισαγώγη στην ομάδα.

Χρήστος Βλάσσης 
ΑΜ: Π2019030

**Comment by DimitrisNtarl** [comment]
Όνομα ομάδας: ionioi
GitHub organisation url: [ionioi](https://github.com/ionioi)
Νέα μέλη δεκτά: ΟΧΙ

| # | ΑΜ | GitHub profile | week 3 |
| -- | -- | -- | -- |
| 1 | 2019215| [gh link](https://github.com/p19ntar) | |
| 2 | 2019135| [gh link](https://github.com/p19biza) | |
| 3 | 2019206| [gh link](https://github.com/Mnlsvt) | |
| 4 | 2019235| [gh link](https://github.com/IsabellaPap) | NO |
| 5 | 2019015| [gh link](https://github.com/alexl14) | |
| 6 | 2019114| [gh link](https://github.com/p19papa4) | |
| 7 | 2018062| [gh link](https://github.com/stovag) | NO |
| 8 | 2019111| [gh link](https://github.com/Genti-voka) | |
| 9 | 2019047| [gh link](https://github.com/Michail-Kavvadias) | |

**Comment by p19tzam** [comment]
Όνομα ομάδας: Kafeneio ☕
GitHub organisation url: [Kafeneio☕](https://github.com/Kafeneio)
Νέα μέλη δεκτά: ΟΧΙ


| # | ΑΜ | GitHub profile | week 3 |
| -- | -- | -- | -- |
| 1 | 2019033 | [gh link](https://github.com/tsimpliarakis/) | |
| 2 | 2019213 | [gh link](https://github.com/p19tzam) | |
| 3 | 2018028 | [gh link](https://github.com/Alkissourvinos) | |
| 4 | 2019193 | [gh link](https://github.com/tsimpliarakis/) | NO |
| 5 | 2019194 | [gh link](https://github.com/makisgrammenos) | |

**Comment by u2nmd** [comment]
Όνομα ομάδας: OompaLoompas
GitHub organisation url: [OompaLoompas](https://github.com/OompaLoompas)
Νέα μέλη δεκτά: ΟΧΙ

| # | ΑΜ | GitHub profile | week 3 |
| -- | -- | -- | -- |
| 1 | 2019034 | [u2nmd](https://github.com/u2nmd) | OK |
| 2 | 2019110 | [p19pasc](https://github.com/p19pasc) | OK |
| 3 | 2019014 | [vasiliskampani](https://github.com/vasiliskampani) | OK |
| 4 | 2019150 | [P2019SARAKATSANIS](https://github.com/P2019SARAKATSANIS) | OK |
| 5 | 2019053 | [AimiliosPavlidis2001](https://github.com/AimiliosPavlidis2001) | OK |
| 6 | 2019086 | [Chris-Livas](https://github.com/Chris-Livas) | OK |
| 7 | 2018171 | [perikarananou](https://github.com/perikarananou) | |
| 8 | 2018012 | [chrislach1](https://github.com/chrislach1) | OK |
| 9 | 2019169 | [TaniaStoupi](https://github.com/TaniaStoupi) | |
| 10 | 2019044 | [p2019044](https://github.com/p2019044) | |
| 11 | 2019107 | [TheodoreZ-107](https://github.com/TheodoreZ-107) | |
| 12 | 2019038 | [p19loli](https://github.com/p19loli) | |

**Reply by vasiliskampani** [reply]
Επιθυμώ να ενταχθώ στην ομάδα 

Ον/νυμο : Βασίλειος-Μπασίρ Καμπάνι
ΑΜ : Π2019014

**Reply by P2019SARAKATSANIS** [reply]
Θα ήθελα να γίνω και εγώ ένα μέλος της ομάδας σας

Ονοματεπώνυμο: Χαράλαμπος Σαρακατσάνης
Α.Μ. : Π2019150

**Reply by u2nmd** [reply]
> Επιθυμώ να ενταχθώ στην ομάδα
> 
> Ον/νυμο : Βασίλειος-Μπασίρ Καμπάνι ΑΜ : Π2019014

Τέλεια, σε προσθέτω άμεσα στην ομάδα!

**Reply by u2nmd** [reply]
> Θα ήθελα να γίνω και εγώ ένα μέλος της ομάδας σας
> 
> Ονοματεπώνυμο: Χαράλαμπος Σαρακατσάνης Α.Μ. : Π2019150

Σε ευχαριστούμε για το αίτημά σου, θα λάβεις σχετική πρόσκληση για συμμετοχή!

**Reply by perikarananou** [reply]
Θα ήθελα να γίνω και εγώ ένα μέλος της ομάδας σας

Περιστέρα Καρανάνου 
Π2018171

**Reply by p2019181** [reply]
Θα ήθελα και εγώ να γίνω μέλος της ομάδας σας 
Μαριόλ Μιέστρι Π2019181

**Reply by AimiliosPavlidis2001** [reply]
Θα επιθυμούσα να εισαχθώ και εγώ στην ομάδα σας. 
Ονοματεπώνυμο: Αιμίλιος Παυλίδης 
Α.Μ: Π2019053

**Reply by Mariadar97** [reply]
θα ήθελα και εγω να γίνω μέλος της ομάδας.
Μαρία Δαραβίγκα 
Π2015076

**Reply by Chris-Livas** [reply]
Επιθυμώ να ενταχθώ στην ομάδα ούμπα λούμπα.
ΛΙΒΑΣ ΧΡΗΣΤΟΣ Π2019086

**Reply by u2nmd** [reply]
> Θα επιθυμούσα να εισαχθώ και εγώ στην ομάδα σας. Ονοματεπώνυμο: Αιμίλιος Παυλίδης Α.Μ: Π2019053

Θα σε προσθέσω άμεσα, ευχαριστούμε για το αίτημα Αιμίλιε!

**Reply by u2nmd** [reply]
> Επιθυμώ να ενταχθώ στην ομάδα ούμπα λούμπα. ΛΙΒΑΣ ΧΡΗΣΤΟΣ Π2019086

Μόλις έγινες ένας ούμπα λούμπα, καλώς ήρθες στην ομάδα Χρήστο!

**Reply by u2nmd** [reply]
> θα ήθελα και εγω να γίνω μέλος της ομάδας. Μαρία Δαραβίγκα Π2015076

Καλησπέρα Μαρία, ευχαριστούμε για το αίτημα εισαγωγής ωστόσο δεν βλέπω να έχεις γράψει κάποια εισαγωγή και να την έχεις ανεβάσει στις [συζητήσεις](https://github.com/courses-ionio/help/discussions) ώστε να εξετάσουμε εάν οι στόχοι σου συμπίπτουν με της ομάδας.

**Reply by u2nmd** [reply]
> Θα ήθελα και εγώ να γίνω μέλος της ομάδας σας Μαριόλ Μιέστρι Π2019181

Καλησπέρα Μαριόλ, οι στόχοι σου στην εισαγωγή σου δεν είναι πολύ ξεκάθαροι τόσο όσο αφορά την ομαδικότητα όσο και τους στόχους που έχεις θέσει. Επομένως, δεν μπορώ να σε εντάξω στην ομάδα.

**Reply by u2nmd** [reply]
> Θα ήθελα να γίνω και εγώ ένα μέλος της ομάδας σας
> 
> Περιστέρα Καρανάνου Π2018171

Καλώς ήρθες στην ομάδα Περιστέρα!

**Reply by chrislach1** [reply]
Θα επιθυμούσα να ενταχθώ στην ομάδα

Ον/νυμο : Λαχανάς Χρήστος
ΑΜ : Π2018012

**Reply by u2nmd** [reply]
> Θα επιθυμούσα να ενταχθώ στην ομάδα
> 
> Ον/νυμο : Λαχανάς Χρήστος ΑΜ : Π2018012

Τέλεια, σε προσθέτω στην ομάδα Χρήστο!

**Reply by TheodoreZ-107** [reply]
Καλησπέρα,
Θα ήθελα να γίνω μέλος της ομάδας
Ον/μο: Ζερβάκης Θεόδωρος
ΑΜ: 2019107

**Reply by TaniaStoupi** [reply]
Καλησπέρα, θα επιθυμούσα να ενταχθώ στην ομάδα σας.
Ονοματεπώνυμο: Στούπη Σουλτάνα - Ανδριάνα
ΑΜ: 2019169

**Reply by p2019044** [reply]
Καλησπέρα σας θα ήθελα να συμμετέχω στην ομάδα σας.
Σκλεπάρης Κωνσταντίνος 
ΑΜ: 2019044

**Reply by u2nmd** [reply]
> Καλησπέρα, θα επιθυμούσα να ενταχθώ στην ομάδα σας. Ονοματεπώνυμο: Στούπη Σουλτάνα - Ανδριάνα ΑΜ: 2019169

Καλησπέρα Τάνια, πολύ ωραία η εισαγωγή σου, σε προσθέτω στην ομάδα μας!

**Reply by u2nmd** [reply]
> Καλησπέρα σας θα ήθελα να συμμετέχω στην ομάδα σας. Σκλεπάρης Κωνσταντίνος ΑΜ: 2019044

Καλησπέρα Κωνσταντίνε, ευχαριστούμε για το αίτημά σου! Θα σε προσθέσω στην ομάδα. Μία παρατήρηση μονάχα, πρόσεξε τα ορθογραφικά λάθη σου.

**Reply by u2nmd** [reply]
> Καλησπέρα, Θα ήθελα να γίνω μέλος της ομάδας Ον/μο: Ζερβάκης Θεόδωρος ΑΜ: 2019107

Καλησπέρα Θοδωρή, έχοντας διαβάσει την εισαγωγή σου πιστεύω ότι αποτελείς μία καλή προσθήκη στην ομάδα. Ευχαριστούμε!

**Reply by Vaso-Lo** [reply]
Καλησπέρα, θα ήθελα να γίνω μέλος της ομάδας. Ονοματεπώνυμο: Βασιλική Λώλη, ΑΜ: Π2019038

**Reply by u2nmd** [reply]
> Καλησπέρα, θα ήθελα να γίνω μέλος της ομάδας. Ονοματεπώνυμο: Βασιλική Λώλη, ΑΜ: Π2019038

Καλησπέρα Βασιλική, μου άρεσε η εισαγωγή σου και θα σε προσθέσω στην ομάδα! Ωστόσο, έχω δύο παρατηρήσεις να σου κάνω. Πρώτον, δεν θα εκτελέσουμε προγραμματιστικές ασκήσεις αλλά μόνο γραμμής εντολών. Δεύτερον, λείπει από την ανάρτησή σου στις συζητήσεις ο σύνδεσμος που οδηγεί στην αναφορά σου.

**Comment by Anastasia-Rizou** [comment]
Όνομα ομάδας: IonianUniverity2019

GitHub organisation url: [IonianUniversity2019](https://github.com/IonianUniversity2019)

Νέα μέλη δεκτά: ΟΧΙ

| # | ΑΜ | GitHub profile |
| -- | -- | -- |
| 1 | 2019156 | [Anastasia-Rizou](https://github.com/Anastasia-Rizou)|
| 2 | 2019005 | [Poulimenos. S](https://github.com/p19poul)|
| 3 | 2018203 | [p18anas](https://github.com/p18anas)|
| 4 | 2019173 | [p19pasa](https://github.com/p19pasa)||
| 5 | 2019127 | [p19verg](https://github.com/p19verg)|
| 6 | 2019013 | [AnastasiaMariaStefani](https://github.com/AnastasiaMariaStefani)|
| 7 | 2018198 | [eliastks09](https://github.com/eliastks09)|
| 8 | 2019223 | [PetroulaStylianiou](https://github.com/PetroulaStylian)|

**Comment by p19karg** [comment]
Όνομα ομάδας: IonianCombination
GitHub organisation url: [IonianCombination](https://github.com/IonianCombination)
Νέα μέλη δεκτά: ΟΧΙ

| # | ΑΜ | GitHub profile | week3 |
| -- | -- | -- | -- |
| 1 | 2019221 | [p19karg](https://github.com/p19karg) | OK |
| 2 | 2019177 | [p19tsik](https://github.com/p19tsik) | OK |
| 3 | 2019021 | [StavroulaAnatolaki](https://github.com/StavroulaAnatolaki)| OK |
| 4 | 2018070 | [Konstantina3](https://github.com/Konstantina3) | |
| 5 | 2018134 | [xristinasp](https://github.com/xristinasp) | |
| 6 | 2018056 | [p18kavv](https://github.com/p18kavv) | |
| 7 | 2018163 | [Vasdekis](https://github.com/Vasdekis) | |
| 8 | 2018105 | [DimitrisTzer](https://github.com/DimitrisTzer) | |

**Comment by ghost** [comment]
Όνομα ομάδας: DevTeamIU
GitHub organisation url: [DevTeamIU](https://github.com/DevTeamIU)
Νέα μέλη δεκτά: ΟΧΙ

| # | ΑΜ | GitHub profile | week3 |
| -- | -- | -- | -- |
| 1 | 2018143 | [Sotirios Koutopoulis](https://github.com/p18kout) | |
| 2 | 2018040 | [Ioannis Kanakis](https://github.com/p18kana) | OK |
| 3 | 2014150 | [Euaggelos Nikolaou](https://github.com/Vagelis1993) | |
| 4 | 2018176 | [Kyriakos Mpaltatzidis](https://github.com/p18balt) | OK |
| 5 | 2019116 | [Rafahl Fwtakidis](https://github.com/rafael5gr2) | |
| 6 | 2018135 | [Elisabet Strabou](https://github.com/initialelizah) | OK |

**Reply by Vagelis1993** [reply]
Καλησπέρα,
Επιθυμώ την ένταξη στην ομάδα σας
Ον/μο: Νικολάου Ευάγγελος
ΑΜ: 2014150

**Reply by ghost** [reply]
Σας προσθέτω στην ομάδα !

**Reply by marioskomianos** [reply]
Καλησπέρα,
Επιθυμώ την ένταξη στην ομάδα
Ον/μο: Κομιανός Μάριος
ΑΜ: 2019231

**Reply by ghost** [reply]
> Καλησπέρα, Επιθυμώ την ένταξη στην ομάδα Ον/μο: Κομιανός Μάριος ΑΜ: 2019231

Σας προσθέτω στην ομάδα

**Reply by p18balt** [reply]
Καλησπέρα , επιθυμώ ένταξη στην ομάδα σας 
Ονοματεπώνυμο: Κυριάκος Μπαλτατζίδης
Αριθμός Μητρώου : Π2018176

**Reply by ghost** [reply]
> Καλησπέρα , επιθυμώ ένταξη στην ομάδα σας Ονοματεπώνυμο: Κυριάκος Μπαλτατζίδης Αριθμός Μητρώου : Π2018176

Σας προσθέτω στην ομάδα.

**Reply by ghost** [reply]
> Καλησπέρα, Επιθυμώ την ένταξη στην ομάδα σας Ον/μο: Νικολάου Ευάγγελος ΑΜ: 2014150

Έχει σταλεί πρόσκληση!

**Reply by ghost** [reply]
> Καλησπέρα, Επιθυμώ την ένταξη στην ομάδα Ον/μο: Κομιανός Μάριος ΑΜ: 2019231

Έχει σταλεί πρόσκληση!

**Reply by ghost** [reply]
> Καλησπέρα , επιθυμώ ένταξη στην ομάδα σας Ονοματεπώνυμο: Κυριάκος Μπαλτατζίδης Αριθμός Μητρώου : Π2018176

Έχει σταλεί πρόσκληση!

**Reply by rafael5gr2** [reply]
Καλησπέρα,
Επιθυμώ την ένταξη στην ομάδα σας
Ονοματεπώνυμο: Φωτακίδης Ραφαήλ
ΑΜ: 2019116

**Reply by initialelizah** [reply]
Καλησπέρα,
Επιθυμώ την ένταξη στην ομάδα σας
Ονοματεπώνυμο: Στραβού Ελισάβετ
ΑΜ: 2018135

**Reply by ghost** [reply]
> Καλησπέρα, Επιθυμώ την ένταξη στην ομάδα σας Ονοματεπώνυμο: Φωτακίδης Ραφαήλ ΑΜ: 2019116

Σας προσθέτω στην ομαδα

**Reply by ghost** [reply]
> Καλησπέρα, Επιθυμώ την ένταξη στην ομάδα σας Ονοματεπώνυμο: Στραβού Ελισάβετ ΑΜ: 2018135

Σας προσθέτω στην ομάδα

**Reply by ghost** [reply]
Παρακαλώ τα μέλη να μου στείλουν ένα μήνυμα στο p18kout@ionio.gr , έτσι ώστε να οργανωθούμε καλύτερα.

**Reply by ghost** [reply]
!!!!! Έχω στείλει invite σε όλα τα μέλη . Παρακαλώ δεχτείτε το αίτημα !

**Reply by ghost** [reply]
Ύστερα απο συζήτηση μεταξύ των μελών ,αποφασίστηκε ομόφωνα στην αποχώρηση του Κομιανού Μάριου για λόγους έλλειψης ενδιαφέροντος και επικοινωνίας με την ομάδα .

**Comment by giannisbarlas1** [comment]


**Reply by vivikara** [reply]
Καλησπέρα επιθυμώ ένταξη στην ομάδα 

Καραγκούνη Παρασκευή 
ΑΜ: Π2017200

**Reply by giannisbarlas1** [reply]
Σας προσθετω στην ομαδα!

**Comment by otolias** [comment]
Όνομα Ομάδας: LostMpodis
Github Organisation URL: [LostMpodis](https://github.com/LostMpodis)
Νέα Μέλη Δεκτά: ΝΑΙ

| # | AM | Github | week 3 |
| -- | -- | -- | -- |
| 1 | 2017095 | [Tolias](https://github.com/p17anto2) | |
| 2 | 2019146 | [Bill](https://github.com/p19poly1) | |
| 3 | 2019067 | [John](https://github.com/john7665) |
| 4 | 2016059 | [Apo](https://github.com/ApoLaz) | |
| 5 | 2016045 | [Stavros](https://github.com/Stavros16) | |
| 6 | 2019009 | [Giannis](https://github.com/giannisbarlas1) | OK |
| 7 | 2016087 | [Alexandros](https://github.com/p16z) | |
| 8 | 2015076 | [Maria](https://github.com/Mariadar97) | OK |
| 9 | 2015167 | [Panos](https://github.com/panosath) | |
| 10 | 2017182 | [Nemanja](https://github.com/IonianIronist) |

**Reply by Stavros16** [reply]
Καλησπέρα,
Επιθυμώ την ένταξη στην ομάδα
Ον/μο: Σταύρος Μαραγγοτίδης
ΑΜ: 2016045

**Reply by otolias** [reply]
Καλησπέρα,
Σου έστειλα invite, ανανεώνω και το προφίλ!

**Reply by giannisbarlas1** [reply]
Καλησπέρα,
Επιθυμώ την ένταξη στην ομάδα
Ον/μο:  Μπαρλας Ιωαννης
ΑΜ: 2019009

**Reply by otolias** [reply]
Καλησπέρα,
Έστειλα invite, κάνω και την ανανέωση

**Reply by p16z** [reply]
Καλησπέρα,
Επιθυμώ την ένταξη στην ομάδα σας
Ον/μο: Αλέξανδρος Ζανέ
ΑΜ: 2016087

**Reply by otolias** [reply]
Καλησπέρα, 
Έστειλα invite.

**Reply by Mariadar97** [reply]
Καλησπερα σας, επιθυμώ να ενταχθλω στην ομαδα 
Μαρία Δαραβίγκα 
ΑΜ: 2015076

**Reply by otolias** [reply]
Καλησπέρα,
έστειλα invite!

**Reply by panosath** [reply]
Καλησπέρα,
Επιθυμώ την ένταξη στην ομάδα.
Ον/μο: Αθανασιάδης Παναγιώτης
ΑΜ: 2015167

**Reply by otolias** [reply]
Καλησπέρα,
έστειλα invite!

**Comment by vuxxs** [comment]
Όνομα Ομάδας: Sybil Reisz

GitHub organization URL: [Sybil Reisz](https://github.com/Sybil-Reisz)

Νέα μέλη δεκτά: ΟΧΙ

| # | ΑΜ | GitHub profile | week 3 |
| -- | -- | -- | -- |
| 1 | 2019118 | [Παναγιώτης Βαμβακάς](https://github.com/Vuxxs) | |
| 2 | 2016172 | [Αναστάσιος Χυτήρης](https://github.com/Combustiblemon) | |
| 3 | 2019027 | [Χρήστος Ζηγγόπης](https://github.com/p19zing) | OK |
| 4 | 2019070 | [Ερρίκος Φοίβος Καββαδίας](https://github.com/p19kavv1) | OK |
| 5 | 2019026 | [Γεώργιος Βρούζος](https://github.com/p19vrou) | OK |

**Comment by mariosconsta** [comment]
Όνομα ομάδας: Cont1nental
GitHub organisation url: [Cont1nental](https://github.com/Cont1nental)
Νέα μέλη δεκτά: ΟΧΙ

| # | ΑΜ | GitHub profile |
| -- | -- | -- |
| 1 | 2015002| [Μάριος Κωνσταντίνου](https://github.com/mariosconsta) |
| 2 | 2014042| [Κοσμάς Τσιανατέλης](https://github.com/Sola1r3) |
| 3 | 2014010| [Μιχαήλ Αχιλλέας Παναγιώτηδης](https://github.com/CptMike) |
| 4 | 2017021| [Γεώργιος Ματθαιόπουλος](https://github.com/p17matt) |
| 5 | 2014074| [Άγγελος Παρλαπανίδης](https://github.com/sosukesama) |
| 6 | 2017186| [Ιωάννης Κωσταρελόπουλος](https://github.com/p17kost) |
| 7 | 2017116| [Αντώνης Κωνσταντόπουλος](https://github.com/2antonisguccigang)|
| 8 | 2015102| [Κωνσταντίνος Βλιώρας](https://github.com/p15vlio)

**Comment by KuhakuNeko** [comment]
Όνομα ομάδας: Git'sTeamEpic
GitHub organisation url: [Git'sTeamEpic](https://github.com/Git-s-PopTeamEpic)
Νέα μέλη δεκτά: ΟΧΙ

| # | ΑΜ | GitHub profile | week 3 |
| -- | -- | -- | -- |
| 1 | 2019108 | [KuhakuNeko](https://github.com/KuhakuNeko) | OK |
| 2 | 2019078 | [Orfeaslambrou](https://github.com/Orfeaslambrou) | |
| 3 | 2019145 | [Geoter711](https://github.com/Geoter711) | |
| 4 | 2019196 | [PeteSKK](https://github.com/PeteSKK) | |
| 5 | 2019059 | [xar1sgeovlacp2019059](https://github.com/xar1sgeovlacp2019059) | |
| 6 | 2019017 | [p19kala](https://github.com/p19kala) | |
| 7 | 2019136 | [gordthrow](https://github.com/gordthrow) | |
| 8 | 2019128 | [alexpoulis](https://github.com/alexpoulis) | |
| 9 | 2018079 | [fireleaf22](https://github.com/fireleaf22) | |

**Comment by ger-lab** [comment]
Όνομα ομάδας: SWkey
GitHub organisation url: [SWkey](https://github.com/SWkey)
Νέα μέλη δεκτά: ΝΑΙ

| # | ΑΜ | GitHub profile |
| -- | -- | -- |
| 1 | 2017005 | [Γιώργος Γεραρχάκης](https://github.com/p17gera) |
| 2 | 2017131 | [Γιώργος Μελιδονιώτης](https://github.com/p17meli) |
| 3 | 2017160 | [Παναγιώτης Σιώλος](https://github.com/p17siol)|
| 4 | 2017200 | [Παρασκευή Καραγκούνη](https://github.com/vivikara)|
| 5 | 2017060 | [Θεόδωρος Γερασόπουλος](https://github.com/p17gera1) |

**Reply by vivikara** [reply]
Καλησπέρα επιθυμώ ένταξη στην ομάδα

Καραγκούνη Παρασκευή
ΑΜ: Π2017200

**Reply by ger-lab** [reply]
Έχουμε στείλει σχετικό invite, κοίταξε στo webmail σου.

**Reply by p17gera1** [reply]
Καλησπέρα παιδιά επιθυμώ να ενταχθώ στην ομάδα σας.


Θεόδωρος Γερασόπουλος με Α.Μ. Π2017060.

**Reply by p17meli** [reply]
Έχει σταλθεί invite στο webmail σου

**Reply by harikoulouri** [reply]
Γεια σας,

Μπορω να μπω στην ομαδας σας παρακαλω, δεν εχω ΑΜ, το ξερει ο καθηγητης μας. 

Ευχαριστω πολυ

**Reply by harikoulouri** [reply]
Γεια σας , μπορω να γινω μελος της ομαδας σας, σας παρακαλω? ευχαριστω

**Reply by p17meli** [reply]
Στάλθηκε invite

**Reply by P17mits1** [reply]
Γεια σας γίνεται να μπω στην ομάδα σας.
Σωτήρης Μήτσος Α.Μ. Π2017087

**Comment by axilleasmandravelis** [comment]
Όνομα ομάδας: ionioteam
GitHub organisation url: [ionioteam](https://github.com/ionioteam)
Νέα μέλη δεκτά: ΟΧΙ

| # | ΑΜ | GitHub profile |
| -- | -- | -- |
| 1 | 2017002 | https://github.com/axilleasmandravelis |
| 2 | 2017178 | https://github.com/p17simt |
| 3 | 2017126 | https://github.com/nikosbout |
| 4 | 2017032 | https://github.com/p17balo |
| 5 | 2017102 | https://github.com/p17papa |

**Comment by P15KOUR** [comment]
Όνομα ομάδας: StudentFossils :t-rex:
 GitHub organisation url: [StudentFossils](https://github.com/StudentFossils/SW/edit/main/Team/README.md)
Νέα μέλη δεκτά: ΟΧΙ

| # | ΑΜ | GitHub profile | 
| -- | -- | -- | 
| 1 | 2015119 | [gh link](https://github.com/P15KOUR) | 
| 2 | 2016132 | [gh link](https://github.com/fanisdimitriadis) | 
| 3 | 2015146 | [gh link](https://github.com/deadoralive1908) | 
| 4 | 2015197 | [gh link](https://github.com/arse21) | 
| 5 |2018093  | [gh link](https://github.com/theoharissiwzos)|

**Comment by diogenisAl** [comment]
Για όποιον/-α: **(1)** δεν έχει ενταχθεί ακόμα σε ομάδα και **(2)** δεν θα προσπαθήσει να εισέλθει σε κάποια από τις ήδη υπάρχουσες, μπορεί να μπει στην παρακάτω, αλλά με **μειωμένο βαθμό**.

Όνομα ομάδας: SWteams
GitHub organisation url: [SWteams](https://github.com/SWteams)

| # | ΑΜ | GitHub profile |
| -- | -- | -- |
| 1 | 2017087 | [P17mits1](https://github.com/P17mits1) |
| 2 | ...| [gh link](...) |
| 3 | ...| [gh link](...) |
| ... | ...| ... |

Απαντήστε στην παρούσα ανάρτηση με τα στοιχεία σας.

**Reply by P17mits1** [reply]
Σωτήρης Μήτσος Α.Μ. Π2017087

**Reply by diogenisAl** [reply]
Καλώς όρισες στην ομάδα @P17mits1!
Υπενθυμίζω ότι σύμφωνα με τους κανόνες, τα μέλη της παρούσας ομάδας θα έχουν μειωμένο βαθμό στις εργασίες τους.
Καλή συνέχεια!

**Reply by agchristos** [reply]
Θα μπορούσα να ενταχθώ στην ομάδα? Αγιοβλασίτης Χρήστος - ΑΜ Π2018089

**Reply by KonstantinosTourtsakis** [reply]
> Θα μπορούσα να ενταχθώ στην ομάδα? Αγιοβλασίτης Χρήστος - ΑΜ Π2018089

Καλησπέρα! Βρίσκεσαι σε λάθος discussion. Το συγκεκριμένο discussion χρησιμοποιήθηκε πέρσι. Μπορείς να αναζητήσεις ομάδα στο [φετινό discussion](https://github.com/courses-ionio/sw/discussions/1133).

**Comment by ghost** [comment]
Όνομα ομάδας: The Dream Team
GitHub organisation url: https://github.com/The-Drem-Team
Νέα μέλη δεκτά: ΟΧΙ

| # | ΑΜ | GitHub profile |
| -- | -- | -- |
| 1 | 2021002 | https://github.com/KyprosAndreou |
| 2 | 2021116 | https://github.com/inf2021116/hci |
| 3 | Inf2021078 | https://github.com/Nek474 |
| 4 | inf2021041 | https://github.com/elenigiannouchou |
| 5 | inf2021170 | https://github.com/Paliaaggeliki |
| 6 | inf2021005 | https://github.com/PanayiotisKl |
